### ISSUE TITLE
Backup schedule and rotation

### ISSUE THREAD

**Issue by franciscopaniskaseker** [litespeedtech/cyberpanel#216]
I have one customer that has cyberpanel (last version) and even with no backups configured on web interface, the backups are happening. An they go untill fill all storage.

This happened when I am trying find/configure backup rotation. When I removed the scheduled backup to create another one, this happened. When I try to "recover" backup, there is no backup to select. When I log as user I can see the logs about created backups, but there is no option or menu to recover backups. Weird.

I tried to create a new scheduled backup and I did not find the "rotation" option. Only the frequence (daily or week).

**So this issue is about:**
1. Is not possible to configure rotation out-of-the-box?
2. How can I identify where is the invible backup rule? I not found anything on crontab.

**Comment by usmannasir** [comment]
What is output of `cat /etc/crontab` ?

**Comment by franciscopaniskaseker** [comment]
```bash
[root@centos-2gb-hel1-1 ~]# cat /etc/crontab
SHELL=/bin/bash
PATH=/sbin:/bin:/usr/sbin:/usr/bin
MAILTO=root

# For details see man 4 crontabs

# Example of job definition:
# .---------------- minute (0 - 59)
# |  .------------- hour (0 - 23)
# |  |  .---------- day of month (1 - 31)
# |  |  |  .------- month (1 - 12) OR jan,feb,mar,apr ...
# |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
# |  |  |  |  |
# *  *  *  *  * user-name  command to be executed

0 * * * * root python /usr/local/CyberCP/plogical/findBWUsage.py
0 * * * * root /usr/local/CyberCP/postfixSenderPolicy/client.py hourlyCleanup
0 0 1 * * root /usr/local/CyberCP/postfixSenderPolicy/client.py monthlyCleanup
0 2 * * * root /usr/local/CyberCP/plogical/upgradeCritical.py
0 12 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/IncBackups/IncScheduler.py Daily
0 0 * * 0 root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/IncBackups/IncScheduler.py Daily
0 3 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/plogical/backupScheduleLocal.py


0 3 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/plogical/backupScheduleLocal.py

0 2 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/plogical/renew.py
[root@centos-2gb-hel1-1 ~]# 

```

So there is an backup rule in the crontab. I not found in /var/spool/cron.

**Comment by usmannasir** [comment]
If you don't want backups to run, please remove these two entries

0 3 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/plogical/backupScheduleLocal.py
0 3 * * * root /usr/local/CyberCP/bin/python2 /usr/local/CyberCP/plogical/backupScheduleLocal.py

**Comment by franciscopaniskaseker** [comment]
Yes, no problem.

How about the backup rotation? Cyberpanel does not has this feature?

**Comment by meramsey** [comment]
Yeah I love this idea and also see we have a feedback suggestion for it already.
https://feedback.cyberpanel.net/b/requests/p/keep-backups-from-last-x-days-purge-old-backups

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.

**Comment by danir-de** [comment]
Still a huge issue. Retention settings are not even rendered in the admin overview and inputs immediately "save" and the clear to show an empty input box when pressing any number key.
